src/core/instance/instance.hpp

```diff
@@ -252,6 +252,16 @@ class Instance : public otInstance, private NonCopyable
      */
     uint32_t GetId(void) const { return mId; }
 
+#if OPENTHREAD_PLATFORM_NEXUS
+    /**
+     * Sets the instance identifier.
+     *
+     * @param[in] aId  The identifier to assign to the `Instance`.
+     *
+     */
```

Comment:
nit: update style here and below
```suggestion
     * @param[in] aId  The identifier to assign to the `Instance`.
     */
```